chaosazure/rgraph/resource_graph.py

```diff
@@ -3,13 +3,25 @@
 
 from chaosazure import auth
 from chaosazure.rgraph.mapper import to_dicts
+from chaoslib.exceptions import FailedActivity
+import msrestazure
+from msrestazure.azure_cloud import AZURE_PUBLIC_CLOUD, \
+    AZURE_US_GOV_CLOUD, AZURE_GERMAN_CLOUD, AZURE_CHINA_CLOUD
 
 
 def fetch_resources(query, resource_type, secrets, configuration):
     with auth(secrets) as cred:
         query = __create_resource_graph_query(
             query, resource_type, configuration)
-        client = ResourceGraphClient(credentials=cred)
+        if secrets.get("azure_cloud") is not None:
```

Comment:
that's seems a repetition of the code __compute_mgmt_client. May I ask why? Could we cleanup?

Reply:
See latest update, `ResourceGraphClient` and `ComputeManagementClient` are moved to `__init__.py`